src/Routes/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Switch, Route} from 'react-router-dom';
+
+import Home from '../components/Home';
+import Header from '../components/Header';
+
+const Routes =()  =>{
+    return(
+    <Switch>
+        <Route exact path ="/" component={Home}/>
+        <Route exact path ='/header' component={Header}/>
```

Comment:
why is this single quote and the above double quote?
are you using a code formatter?

Reply:
done

Reply:
fixed